R/primary-keys.R

```diff
@@ -1,18 +1,18 @@
-# for external users: also checks if really is primary key
+# for external users: also checks if a column really is primary key
 
 #' Mark a column of a table in a [`dm`] object as its primary key
 #'
-#' @description `cdm_add_pk()` marks the given column as the given table's primary key
-#' in the `data_model`-part of the [`dm`] object. If `check == TRUE`, it also first checks if
-#' the given column is a unique key of the table. If `force == TRUE`, it replaces an already
+#' @description `cdm_add_pk()` marks the specified column as the primary key of the specified table
+#' in the `data_model`-part of the [`dm`] object. If `check == TRUE`, then it will first check if
```

Comment:
We no longer have a "data model part". Can you please adapt here and everywhere? Any occurrence of "data_model" is likely wrong.